### DISCUSSION TITLE
Casks don't auto launch ~ Lulu, Rectangle, Bartender (Big Sur)

### DISCUSSION THREAD

**Discussion by huzzam** [Homebrew/discussions#2355] [Casks]
Three of the casks I've installed don't auto launch: Lulu, Bartender, and Rectangle. All three have "Launch on Login" settings, but none of them launch with that setting. I've also tried manually adding them to my Login Items, and they don't launch that way either.

Meanwhile, I downloaded them from their respective sites, and manually installed, and all three work perfectly, launching on login as they should (without needing to add them to Login Items).

I'm using stock settings for Homebrew, no customizations, and I'm on Big Sur 11.6. This is on two systems, an M1 Macbook Air and an Intel iMac i9.

**Comment by SMillerDev** [comment]
If you open them manually after installing from Homebrew, do they open normally?

**Reply by huzzam** [reply]
yes, I can manually launch them just fine, and they work.
